Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```
@@ -116,9 +116,9 @@
   description: Perfect for dealing with paperwork and matters with bureaucracy.
   components:
   - type: Sprite
-    sprite: Clothing/Hands/Gloves/hop.rsi
+    sprite: _Harmony/Clothing/Hands/Gloves/hop.rsi
   - type: Clothing
-    sprite: Clothing/Hands/Gloves/hop.rsi
+    sprite: _Harmony/Clothing/Hands/Gloves/hop.rsi
```

```suggestion
    sprite: _Harmony/Clothing/Hands/Gloves/hop.rsi # Harmony change. Resprite
```

Upstream file changes require a comment.